src/main/menu.js

```diff
@@ -80,16 +81,27 @@ function menuTemplate (ipc) {
             logger.info('[MENU] Import Configuration', result)
             if (result.canceled) return
             if (!result.filePaths || !result.filePaths.length) return
-            userConfig.importSettings(result.filePaths[0], (err) => {
+            userConfig.importSettings(result.filePaths[0], (err) => 
+            {
               if (err) return onerror(err)
-              ipc.send('reload-config', (err) => {
-                if (err) logger.error(err)
-                logger.debug('[SYSTEM] Forcing window refresh')
+              ipc.send('reload-config', async (err) => 
+              {
+                if (err) 
+                {
+                  logger.error(err)
+                  logger.debug('[SYSTEM] Forcing window refresh')
+                  focusedWindow.webContents.send('force-refresh-window')
+                  logger
```

Comment:
need to remove this line since it's causing lint errors

Reply:
still need to remove this line

Reply:
What is the linting error here. I didn't add this, I just changed the formatting, and thats why it looks I made changes

Reply:
OHH nvm I see it

Reply:
line 85 has a standalone line that you added of just `logger`